pkg/cmd/kind/build/nodeimage/nodeimage.go

```diff
@@ -68,7 +68,7 @@ func NewCommand(logger log.Logger, streams cmd.IOStreams) *cobra.Command {
 	cmd.Flags().StringVar(
 		&flags.KubeRoot, "kube-root",
 		"",
-		"path to the Kubernetes source directory (if empty, the path is autodetected)",
+		"DEPRECATED: please switch to just the argument. path to the Kubernetes source directory (if empty, the path is autodetected)",
```

Comment:
```suggestion
		"DEPRECATED: please switch to just the argument. Path to the Kubernetes source directory (if empty, the path is autodetected)",
```

Reply:
Thank you to comment!
Exactly yes. :+1: 
I've commited your suggestion.